bitsail-connectors/connector-redis/src/main/java/com/bytedance/bitsail/connector/redis/config/RedisOptions.java

```diff
@@ -42,7 +42,7 @@ public class RedisOptions implements Serializable {
   private int ttlInSeconds;
 
   /**
-   * Batch send by pipeline after 'batchInterval' records.
+   * Also BatchSize, Batch send by pipeline after 'batchInterval' records.
```

Comment:
still not 100% get this. is this the number of record per batch? and we send one batch through one pipeline? and this mean how many records we put into one batch?

Reply:
we send one batch in the pipelines, I describe it wrongly in the doc, this is just the number of record per batch.

Reply:
I modify it to 
![image](https://user-images.githubusercontent.com/47937942/200501015-ca4363cc-ae24-4d11-8a88-436761f86d0b.png)